tst/suites/parallel.suite

```diff
@@ -14,14 +14,11 @@
 # parallel suite
 
 advection/advection_mpi
-binary/binary_mpi
-binary_adi/binary_adi_mpi
```

Comment:
If due to time constraints, do we think these are the best tests to eliminate for overall coverage? Or were these selections moreso made due to the time the test took?

Reply:
These are really nice tests but they are very slow on github runners. The GPU CI runs the serial versions of these, we may want to also have a darwin CPU CI to run expensive CPU MPI tests as well